### DISCUSSION TITLE
Text in NetBeans looks like gibberish

### DISCUSSION THREAD

**Discussion by anitaymoore** [apache/netbeans#5664] [General]
Okay, I know it's NOT gibberish, but it's definitely not in English and I am really struggling with this.

FYI, I am new to all of this...I mean really new, be kind. I've installed NetBeans 16 along with JDK-16.0.1 but something is definitely not right and I just cannot seem to find the answer. I'm sure someone has asked this, I'm not the only one experiencing this...but I just can't find what I need. Below is a screenshot of what I see...

![NB16](https://user-images.githubusercontent.com/5271248/225434815-702cf056-5183-48be-8b4d-0825e68ce609.png)

If anyone knows how to fix it and is willing to give me a very step-by-step path to help me, I would very much appreciate it.

Thanks in advance,
Anita

**Comment by mbien** [comment]
interesting :)

This might be a broken font, since some words are actually readable.

what operating system is this? And how did you install NetBeans?

**Reply by anitaymoore** [reply]
Thank you so much for that. I'd been staring at this so long I wasn't even thinking of going into the options any longer...a simple solution right in front of my face and I couldn't see the forest for the trees. It may have been a broken font (that was an excellent suggestion), but when I selected a different font that didn't fix it. However, you put me on the right thinking path and I decided to check out the themes in the Look & Feel. When I went into Tools/Options/Look and Feel and selected a different theme for NetBeans, that did the trick! I even went back in and went back to the original Nimbus theme and it was still fixed, so simply selecting a different theme forced a restart of the application that reconnected something that was broken.

In case anyone else has this issue...I also experienced this issue in NetBeans 8.1 (my original install) and NetBeans 16, this same fix worked for both. I had originally installed 8.1 because that's what a previous developer had used.

Thanks for helping me out!!

**Reply by mbien** [reply]
good to hear that you found a solution.

I would be still curious though:
what operating system and what themes (what worked and what did not) are we talking about?

as a side note:
you mentioned JDK-16.0.1 which is actually end of life now. It is better to run latest NetBeans on JDK 17 which is a long term release and will receive updates for a while, or you pick any other JDK versions which are mentioned on the download page. But this would probably not solve your problem thats why I didn't mention it right away.
